spec/controllers/restaurants_controller_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+require 'pry'
+
+RSpec.describe RestaurantsController, type: :controller do
+
+  describe 'GET #index' do
+    it 'populates an array of restaurants' do
+      restaurant = FactoryGirl.create(:restaurant)
+      get :index
+      assigns(:restaurants).should eq([restaurant])
+    end
+
+    it "renders the :index view" do
+      get :index
+      response.should render_template :index
+    end
+ end
+
+describe 'GET #show' do
```

Comment:
Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.